standards/decommissioning.md

```diff
@@ -1,23 +1,42 @@
-* update related documentation where the app/codebase's name comes up. The app's name can be searched in NYPL's Confluence wiki or by asking a teammate in Digital.
+* let the team know that an app/codebase is deprecated or will be deprecated on a specific date.
+  * A message SHOULD be sent in NYPL's `#dev` Slack channel or email `digitaldev@nypl.org`.
+* update related documentation where the app or codebase's name is referenced. The app's name SHOULD be searched in NYPL's Confluence site and the `NYPL`, `NYPL-Simplified`, and `NYPL-Discovery` Github organizations.
+* update the codebase's README and submit a PR.
+  * This MUST include a "Deprecated" heading at the top, a "no longer supported" note, and the date of decommission.
+  * If the repository has moved to a different source control location, the link MUST be included.
 * archive the repo on Github.
-* make sure that all the application servers are shutdown. This includes the development, qa, and production servers on AWS Elastic Beanstalk and any that NYPL IT hosts.
+* work with IT make sure that all the AWS EBS, ECS, or Lambda servers are shutdown. This includes the QA and Production servers in their relevant AWS accounts.
 
-## Example
+## Team-specific Processes
 
-In 2020, the Staff Picks react application was decommissioned and moved to a Drupal-based application. The following is on the [Staff Picks README](https://github.com/nypl/staff-picks).
+The above serves as a general guideline for decommissioning and deprecating products, services, and codebases. Teams can have their own process, in addition to the above, that is more detailed and specific to the type of products they work on. Regardless of all the steps involved, it's important to communicate it and update documentation.
 
-```
+The LSP team has to decommission certain endpoints and services and it was captured in their `lsp_workflows` [Deprecating a Service](https://github.com/NYPL/lsp_workflows/pull/24/files) document.
+
+## Examples
+
+### Application Restructure
+
+In 2020, the Staff Picks React application was decommissioned and moved to a Drupal-based application. The following is on the [Staff Picks README](https://github.com/nypl/staff-picks).
+
+```md
 # Deprecated
 As of February 2020, this app is no longer in use at NYPL, and the current implementation is Drupal-based. Please contact the DXP/RENO team in Digital for more information. For NYPL developers, please see documentation on the update on [Confluence](https://confluence.nypl.org/pages/viewpage.action?pageId=24150584).
 ```
+
+### Lambda Replatform
+
+#### BarcodeService
+
+The [BarcodeService](https://github.com/NYPL/barcode-service) app was a Node-wrapped PHP AWS Lambda that served the Platform API `/api/v0.1/patron/{id}/barcode` endpoint. The codebase and lambda have not been updated since 2017 and it was verified that the lambda had no invocations in the past 12 months.
```

Comment:
```suggestion
The [BarcodeService](https://github.com/NYPL/barcode-service) app was a Node-wrapped PHP AWS Lambda that served the Platform API `/api/v0.1/patron/{id}/barcode` endpoint. The codebase and lambda had not been updated since 2017, and it was verified that the lambda had no invocations in the previous 12 months.
```